plone/app/locales/locales/pt_BR/LC_MESSAGES/plone.po

```diff
@@ -9729,7 +9727,7 @@ msgstr "Local do evento"
 #. Default: "This event is open ended."
 #: plone.app.event/plone/app/event/dx/behaviors.py:133
 msgid "help_event_open_end"
-msgstr ""
+msgstr "Final aberto."
```

Comment:
Não deveria ser "Esse evento está aberto"?

Reply:
I'm not sure about this.